src/molecules/tanstack-table/tanstack-table.stories.tsx

```diff
@@ -19,6 +19,16 @@ Default.args = {
   columns: tanstackTableCreateColumnsByRowData({
     row: users[0],
     languageData: { userName: 'Kullanıcı Adı' },
+    links: {
+      userName: {
+        targetAccessorKey: 'id',
+        prefix: 'http://192.168.1.105:1453/tr/app/admin',
+        suffix: '/edit',
+      },
+      email: {
+        prefix: 'http://192.168.1.105:1453/tr/app/',
+      },
+    },
```

Comment:
I feel we can have a direct way to change the cells instead of focusing on links only/

Reply:
In next pr we will able to show icon. Do you mean something other than this?